apps/nextjs-website/src/components/organisms/Auth/LoginForm.tsx

```diff
@@ -1,19 +1,168 @@
 'use client';
-import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react';
+import { translations } from '@/_contents/translations';
+import { useAuthenticator } from '@aws-amplify/ui-react';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import {
+  Box,
+  Typography,
+  Stack,
+  Checkbox,
+  Grid,
+  TextField,
+  Divider,
+  Button,
+  Card,
+  FormControlLabel,
+  FormControl,
+  InputLabel,
+  OutlinedInput,
+  InputAdornment,
+  IconButton,
+  Snackbar,
+  Alert,
+  useTheme,
+} from '@mui/material';
+import { IllusLogin } from '@pagopa/mui-italia';
+import { Auth } from 'aws-amplify';
+import Link from 'next/link';
 import { redirect } from 'next/navigation';
+import { MouseEvent, useCallback, useState } from 'react';
 
 const LoginForm = () => {
+  const {
+    auth: { login },
+    shared,
+  } = translations;
+  const {
+    palette: { primary },
+  } = useTheme();
+
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
   const { authStatus } = useAuthenticator((context) => [context.authStatus]);
 
   if (authStatus === 'authenticated') {
     redirect('/');
   }
 
+  const handleClickShowPassword = useCallback(
+    () => setShowPassword((show) => !show),
+    []
+  );
+
+  const handleMouseDownPassword = useCallback(
+    (event: MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+    },
+    []
+  );
+
+  const onLogin = useCallback(() => {
+    Auth.signIn(username, password).catch((e) => setError(e.message));
```

Comment:
The best solution would be a function that given the `e.code` returns the error message obtained from the `translations` file rather than displaying the message provided by the backend. But I believe the [DEV-1047](https://pagopa.atlassian.net/jira/software/projects/DEV/boards/280?selectedIssue=DEV-1047) will be used to do this.

Reply:
Yes, this is just to let the user knows if there's an error